Sources/Purchasing/Purchases.swift

```diff
@@ -1977,10 +1984,12 @@ private extension Purchases {
         updateAllCachesIfNeeded()
         dispatchSyncSubscriberAttributesIfNeeded()
 
+#if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
```

Comment:
same thinking re: skipping catalyst

Reply:
FWIW we have ios + maccatalyst checks elsewhere -- `showPriceConsent` being one place..idk if we want to get those consistent?

Reply:
it'd be great to establish clear rules for them, especially since they're mostly up to personal preference. Feel free to go with whichever you prefer for now